Tasks/DownloadPackage/download.ts

```diff
@@ -50,8 +50,9 @@ export async function downloadPackage(collectionUrl: string, credentialHandler:
 	
```

Comment:
will failure in this call result in failure of the task?
vsoClient.getVersioningData

do we need to set tl.setResult upon failure of this call?